src/pas/plugins/oidc/plugins.py

```diff
@@ -215,6 +217,8 @@ def rememberIdentity(self, userinfo):
                                 # https://bandit.readthedocs.io/en/1.7.4/plugins/b110_try_except_pass.html
                                 pass
                             self._updateUserProperties(user, userinfo)
+                            if user is not None:
+                                notify(PrincipalCreated(user))
```

Comment:
Is notifying this enough? For instance, when a user logs in a Plone site and Plone fires portal_membership's `loginUser`, this tool notifies either `UserInitialLoginEvent` or `UserLoggedInEvent`.

I think that we should be notifying those two events too.

Reply:
If I see it correctly, it can also happen that member properties have to be updated with every new login. From this point of view, it definitely makes sense to notifying more events.

Reply:
Nice catch indeed.